packages/ra-ui-materialui/src/layout/TitlePortal.tsx

```diff
@@ -3,13 +3,18 @@ import { Typography, TypographyProps } from '@mui/material';
 
 export const TitlePortal = (props: TypographyProps) => (
     <Typography
-        flex="1"
-        textOverflow="ellipsis"
-        whiteSpace="nowrap"
-        overflow="hidden"
         variant="h6"
         color="inherit"
         id="react-admin-title"
         {...props}
+        sx={[
+            {
+                flex: '1',
+                textOverflow: 'ellipsis',
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
+            },
+            ...(Array.isArray(props.sx) ? props.sx : [props.sx]),
+        ]}
```

Comment:
Same here, do we really need the Array check?

Reply:
Yes, SxProps can be functions, which we cannot directly unpack